blockonomics-woocommerce.php

```diff
@@ -401,14 +401,62 @@ function bnomics_async_scripts($url)
 register_activation_hook( __FILE__, 'blockonomics_activation_hook' );
 add_action('admin_notices', 'blockonomics_plugin_activation');
 
+global $blockonomics_db_version;
+$blockonomics_db_version = '1.0';
+
 function blockonomics_activation_hook() {
     if(!is_plugin_active('woocommerce/woocommerce.php'))
     {
         trigger_error(__( 'Wordpress Bitcoin Payments - Blockonomics requires WooCommerce plugin to be installed and active.', 'blockonomics-bitcoin-payments' ).'<br>', E_USER_ERROR);
     }
+
+    // Create blockonomics_orders table
+    // https://codex.wordpress.org/Creating_Tables_with_Plugins
+    global $wpdb;
+    global $blockonomics_db_version;
+
+    $installed_ver = get_option( "blockonomics_db_version" );
+
+    if ( $installed_ver != $blockonomics_db_version ) {
+        $table_name = $wpdb->prefix . 'blockonomics_orders';
+        $charset_collate = $wpdb->get_charset_collate();
+        $sql = "CREATE TABLE IF NOT EXISTS $table_name (
+            order_id int NOT NULL,
+            status int NOT NULL,
+            crypto varchar(3) NOT NULL,
+            address varchar(191) NOT NULL,
+            timestamp int,
+            time_remaining int,
+            satoshi int,
+            currency varchar(3),
+            value longtext,
+            txid text,
+            UNIQUE (address),
```

Comment:
Unique keyword is unnecessary. We already are using PRIMARY KEY

Reply:
Updated in 38893c1 and fa71700